src/patternfly/components/DescriptionList/description-list.scss

```diff
@@ -86,13 +103,20 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "md",
 .pf-c-description-list__term {
   font-size: var(--pf-c-description-list__term--FontSize);
   font-weight: var(--pf-c-description-list__term--FontWeight);
+  line-height: var(--pf-c-description-list__term--LineHeight);
 
   .pf-c-description-list__text {
     display: inline;
   }
 }
 
+.pf-c-description-list__description {
+  line-height: var(-pf-c-description-list__description--LineHeight);
```

Comment:
```suggestion
  line-height: var(--pf-c-description-list__description--LineHeight);
```

Reply:
Also if we leave the font-size as it is (16px) and use a small line-height, the text is going to look cramped. I think the small line height is mostly for use with small text, or text that doesn't wrap.

Here's a small line-height:

<img width="827" alt="Screen Shot 2021-07-28 at 5 06 51 PM" src="https://user-images.githubusercontent.com/35148959/127402485-1d2fae1f-80cb-4a8f-ab74-c21e17d9c5f0.png">

regular line-height

<img width="838" alt="Screen Shot 2021-07-28 at 5 07 00 PM" src="https://user-images.githubusercontent.com/35148959/127402525-02a3f9e6-b4f3-4c2d-beee-18d202dd3b95.png">

wdyt @mattnolting

Reply:
ping @mattnolting @mcarrano @mceledonia wdyt about this change? It's the new default.

### Existing - regular line height for term and description
<img width="837" alt="Screen Shot 2021-08-05 at 10 22 40 AM" src="https://user-images.githubusercontent.com/35148959/128376871-eeb8db7e-7272-4652-a57b-4b6ee13026f0.png">

### In this PR - small line height for term and description
<img width="860" alt="Screen Shot 2021-08-05 at 10 22 53 AM" src="https://user-images.githubusercontent.com/35148959/128376886-3e79ba25-0f67-4787-b8b0-1bba2852933e.png">

### FWIW, what it would look like with small line height on term (which has a small font size), and regular line height on description (has regular font size)
<img width="855" alt="Screen Shot 2021-08-05 at 10 23 07 AM" src="https://user-images.githubusercontent.com/35148959/128376901-006a7d11-559c-4bd5-8ee8-b07f5c9c09f6.png">